src/components/Content/Pursuance/views/DiscussView.js

```diff
@@ -5,22 +5,32 @@ import ReactMarkdown from 'react-markdown';
 import FaEllipsisV from 'react-icons/lib/fa/ellipsis-v';
 import FaCircleO from 'react-icons/lib/fa/circle-o';
 import './DiscussView.css';
+import { getTasks } from '../../../../actions';
 
 const leapChatUrl = "http://localhost:8080/#GiddinessPuttRegisterKioskLucidityJockstrapTastebudFactoryPegboardOpticalEstrogenGoatskinHatchlingDittoPseudoNegotiatorLunchboxLightbulbUploadSyllableTulipQuiltJurorRuptureAorta";
 
 class DiscussView extends Component {
 
+  componentWillMount(){
+    const { match: { params: { pursuanceId } }, tasks, getTasks } = this.props;
+    if (!tasks.taskMap['1_1']) {
+      getTasks(pursuanceId);
+    }
+  }
+
   render() {
     const { pursuances, tasks } = this.props;
     // TODO: Un-hardcode after demo
-    const taskGid = '1_58';
-    console.log('tasks.taskMap', tasks.taskMap);
+    const taskGid = '1_1';
     const task = tasks.taskMap[taskGid];
+    if(!task){
+      return <div>Ain't nobody got task fo' that.</div>
+    }
     const assignedPursuanceId = task.assigned_to_pursuance_id;
     return (
-      <div id="discuss-ctn">
-        <iframe id="leapchat-frame" title="Leapchat" src={leapChatUrl} />
-        <div className="task-details-ctn">
+      <div className="discuss-ctn">
+        <iframe className="leapchat-frame" title="Leapchat" src={leapChatUrl} />
+        <div className="task-description-ctn">
```

Comment:
@moeshoman Please revert this back to the `details` version

Reply:
@Moe-Shoman Thanks :+1:

Reply:
Ah is this changed in the latest? Sorry, missed this

Reply:
All good